tests/usbredir_test.go

```diff
@@ -175,10 +177,17 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 	})
 })
 
-func createVMI(virtClient kubecli.KubevirtClient, enableUsbredir bool) (*v1.VirtualMachineInstance, error) {
-	randomVmi := tests.NewRandomVMI()
-	if enableUsbredir {
-		randomVmi.Spec.Domain.Devices.ClientPassthrough = &v1.ClientPassthroughDevices{}
-	}
+func createVMIWithUsbredirEnabled(virtClient kubecli.KubevirtClient) (*v1.VirtualMachineInstance, error) {
+	randomVmi := libvmi.New(
+		libvmi.WithResourceMemory("1Mi"),
+		libvmi.WithClientPassthrough(),
+	)
+	return virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(randomVmi)
+}
+
+func createVMIWithoutUsbredirEnabled(virtClient kubecli.KubevirtClient) (*v1.VirtualMachineInstance, error) {
+	randomVmi := libvmi.New(
+		libvmi.WithResourceMemory("1Mi"),
+	)
 	return virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(randomVmi)
```

Comment:
We can also check for errors here and avoid returning an error from this func

Reply:
Done.